core/src/main/java/hudson/PluginManager.java

```diff
@@ -446,6 +478,40 @@ public void run(Reactor reactor) throws Exception {
         }});
     }
 
+    @Extension
+    public final static PluginManagerAdministrativeMonitor NOTICE = new PluginManagerAdministrativeMonitor();
```

Comment:
No reason to have a public static field for this. To add, you could look up the extension instance from the extension list.